less/bootstrap/.editorconfig

```diff
@@ -0,0 +1,17 @@
+# editorconfig.org
```

Comment:
I think we'll just need the `.less` files and the LICENSE, unless I'm missing something else you know of. It'd be great if you could amend the commit to remove the files we don't need to store in our repository - 12MB is kind of huge. :)

Reply:
Yup, you can remove the js folder, react-bootstrap completely replaces that.

You shouldn't need those files locally either. If it turns out you do need something (I'm not sure about the fonts, actually, if you plan on using those - maybe you can look into that), definitely keep it in the repository. But the rest can be removed from the repo and locally.

Reply:
Okay, done. Kept the fonts folder as of now, just in case. Ran into a little git mess, trying to delete this specific commit and all. But a learning process, all good. :)